### PR TITLE
replace options api for composition api

### PR DESCRIPTION
**Pull request by joebrock666**

### CODE REVIEW

File: src/views/Votes.vue [hunk 1]
```diff
@@ -125,128 +125,155 @@ import AppToolbarCentered from '@/components/AppToolbarCentered'
...
-      if (!this.validateVotes()) {
+    }
+    const upVote = (id) => {
+      store.commit('delegates/upVote', id)
+    }
+    const downVote = (id) => {
+      store.commit('delegates/downVote', id)
+    }
+    const sendVotes = () => {
+      if (!validateVotes()) {
```
Comment (bludnic) [inline src/views/Votes.vue:201]: The `validateVotes()` is called before declaration. Swap `sendVotes` and `validateVotes` function declarations.

File: src/views/Votes.vue [hunk 2]
```diff
@@ -125,128 +125,155 @@ import AppToolbarCentered from '@/components/AppToolbarCentered'
 import Pagination from '@/components/Pagination'
 import { explorerUriToOnion } from '@/lib/uri'
 import DelegatesTable from '@/components/DelegatesTable/DelegatesTable'
+import { computed, onMounted, ref, reactive } from "vue"
+import { useStore } from "vuex"
+import { useI18n } from 'vue-i18n'
 
 export default {
```
Comment (bludnic) [inline src/views/Votes.vue:132]: defineComponent

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/views/Votes.vue (hunk 2 above)
Comment (joebrock666) [reply src/views/Votes.vue]: Alright, so I should use defineComponent function for both views and components?

File: src/views/Votes.vue [hunk 3]
```diff
@@ -125,131 +125,153 @@ import AppToolbarCentered from '@/components/AppToolbarCentered'
...
+      totalVotes,
+      pages,
+      showPagination,
+      reviewButtonDisabled,
+      getExplorerUrl,
+      upVote,
+      downVote,
+      sendVotes,
+      validateVotes,
+      showConfirmationDialog
```
Comment (bludnic) [inline src/views/Votes.vue:271]: Need to return `VOTE_REQUEST_LIMIT ` because it is used inside the `<template/>`

File: src/views/Votes.vue (hunk 2 above)
Comment (bludnic) [reply src/views/Votes.vue]: Yes.

`defineComponent` is just a type helper for TypeScript. It doesn't do anything.

**Review by bludnic — CHANGES_REQUESTED** [review]

**Pull request merged**